src/BullaFrendLend.sol

```diff
@@ -197,6 +222,9 @@ contract BullaFrendLend is BullaClaimControllerBase {
         Claim memory claim = _bullaClaim.getClaim(claimId);
         _checkController(claim.controller);
 
+        // Calculate total amount due including interest
+        uint256 totalDue = getTotalAmountDue(claimId);
```

Comment:
this can be `calculateCurrentInterest` because claim.claimAmount is already available here